README.md

```diff
@@ -1,24 +1,28 @@
 # The Orange Alliance Node.js
+
 The Orange Alliance Library for Node.js
+
 > Note: Only supports the GET api currently.
 
+![GitHub issues](https://img.shields.io/github/issues/orange-alliance/the-orange-alliance-nodejs?style=for-the-badge)
+
 ## Usage
 
 Using the API is simple. Just create an instance of the API class, passing in your auth key and the name of your app.
 From there, call any of the methods to retrieve the data from the API.
 
 ```ts
-import API from 'toa-api';
+import API from "toa-api";
```

Comment:
It should be ``{ API }`` and ``@the-orange-alliance/api``

Reply:
All set in 5a2c0e